svcdiscovery/plugin/svcdiscovery/plugin.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2020 SUSE
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package svcdiscovery
+
+import (
+	"context"
+	"net"
+
+	"github.com/coredns/coredns/plugin"
+	clog "github.com/coredns/coredns/plugin/pkg/log"
+	"github.com/coredns/coredns/request"
+	"github.com/miekg/dns"
+)
+
+// ServiceDiscovery is a CoreDNS plugin for handling Cloud Foundry App Service
+// Discovery.
+type ServiceDiscovery struct {
+	Next plugin.Handler
+
+	log       clog.P
+	sdcClient *SDCClient
+	ttl       uint32
+}
+
+// Name returns the plugin name.
```

Comment:
```suggestion
// Name of the plugin.
```
(We _don't_ need to always stick to `… returns` for getters)

Presumably this is to satisfy the `plugin.Handler` interface? If yes, we should document that.

Reply:
Fixed with 20607dd.